lib/internal/fs/sync_write_stream.js

```diff
@@ -23,7 +23,12 @@ ObjectSetPrototypeOf(SyncWriteStream.prototype, Writable.prototype);
 ObjectSetPrototypeOf(SyncWriteStream, Writable);
 
 SyncWriteStream.prototype._write = function(chunk, encoding, cb) {
-  writeSync(this.fd, chunk, 0, chunk.length);
+  try {
+    writeSync(this.fd, chunk, 0, chunk.length);
+  } catch (e) {
+    cb(e);
+    return true;
```

Comment:
```suggestion
    return;
```

I've removed the other `return` statement in https://github.com/nodejs/node/commit/d09ff0db705ec17edd939080de3c18758e628787